near-sdk/src/store/lookup_map/mod.rs

```diff
@@ -18,17 +16,12 @@ const ERR_ELEMENT_DESERIALIZATION: &str = "Cannot deserialize element";
 const ERR_ELEMENT_SERIALIZATION: &str = "Cannot serialize element";
 const ERR_NOT_EXIST: &str = "Key does not exist in map";
 
-type LookupKey = [u8; 32];
+type LookupKey = Vec<u8>;
```

Comment:
> Prefixed storage to make it easier to delete (can also have this with the hash with only 39a6e81, but this defeats the purpose of the hash not allocating)

Maybe we can allow the `LookupKey` to be parameterized in `LookupMap` to allow users to either choose this current lookup key or the hashing version, but that might be a bit unwieldy since I could see it adding a lot more type parameters